src/api/question/index.js

```diff
@@ -0,0 +1,15 @@
+export default {
+    // 后台
+    // 获取类型名
+    'getTypeName': '/admin/get-type-name',
```

Comment:
```suggestion
    'getQuestionTypes': '/admin/get-question-types',
```
这个 `get-type-name` 含义太广泛，建议你们后端也要改一下